File: python/ops/lekcija 22/ekran_telefona.py
import math

def EkranTelefona(sirina, visina):
    return {
        'sirina'    : sirina,
        'visina'    : visina
        }

def dijagonala(e):
    if type(e) != dict:
        return -1

    for kljuc in e.keys():
        if kljuc not in ['sirina', 'visina']:
            return -1

    return math.hypot(e['sirina'], e['visina'])

File: java/us-ojp/video17/GrupaA/src/grupaa/EkranTelefona.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package grupaa;

/**
 *
 * @author jasarsoft
 */
public class EkranTelefona {
    private double sirina, visina;

    public double getSirina() {
        return sirina;
    }

    public double getVisina() {
        return visina;
    }

    public EkranTelefona(double sirina, double visina) {
        this.sirina = sirina;
        this.visina = visina;
    }
    
    public double getDiagonala() {
        return Math.sqrt(Math.pow(sirina, 2.0) + Math.pow(visina, 2.0));
    }
    
    public static void zamjeniMjesta(EkranTelefona a, EkranTelefona b) {
        double tmp = a.sirina;
        a.sirina = b.sirina;
        b.sirina = tmp;
        
        tmp = a.visina;
        a.visina = b.visina;
        b.visina = tmp;
    }
}
